packages/web/components/DrawerItems.tsx

```diff
@@ -0,0 +1,81 @@
+import MetaCalendar from 'assets/drawer/calendar.png';
+import MetaDiscord from 'assets/drawer/discord.png';
+import MetaGithub from 'assets/drawer/github.png';
+import MetaGuilds from 'assets/drawer/guilds.png';
+import MetaLeaderboard from 'assets/drawer/leaderboard.png';
+import MetaLibrary from 'assets/drawer/library.png';
+import MetaView from 'assets/drawer/metaview.png';
+import MetaNewsletter from 'assets/drawer/newsletter.png';
+import MetaSeedmarket from 'assets/drawer/seedmarket.png';
+import MetaTwitter from 'assets/drawer/twitter.png';
+
+export interface DrawerItemType {
+  href: string;
+  isExternal?: boolean;
+  src: string;
+  alt: string;
+}
+
+export const DrawerItems = [
```

Comment:
This isn't a component, maybe move that file somewhere else
(and does not contain JSX => `.ts`)